src/Entity/Security/LocalAccount.php

```diff
@@ -392,47 +279,19 @@ public function setPasswordRequestedAt(\DateTime $date = null)
     /**
      * Gets the timestamp that the user requested a password reset.
      *
-     * @return \DateTime|null
+     * @return Datetime | null
      */
     public function getPasswordRequestedAt()
     {
         return $this->passwordRequestedAt;
     }
 
-    /**
-     * isPasswordRequestNonExpired
-     * Insert description here.
-     *
-     * @param $ttl
-     *
-     * @return
-     *
-     * @static
-     *
-     * @see
-     * @since
-     */
     public function isPasswordRequestNonExpired($ttl)
     {
         return null === $this->getPasswordRequestedAt() || (
-               $this->getPasswordRequestedAt() instanceof \DateTime &&
```

Comment:
Why is this line removed? The next line assumes the result of `$this->getPasswordRequestedAt()` is not null, and this check is here to make sure of that.

Reply:
I think this was because phpstan was telling me it would always evaluate to true
`Instanceof between DateTime and DateTime will always evaluate to true.`
Shamefully, I just accepted what the machine told me. Now that I checked it's clear `getPasswordRequestedAt` can also return null.

tldr: line is back